utils/getOffchainPrice.ts

```diff
@@ -57,7 +61,7 @@ export const PRICEFEED_PARAMS: PricefeedParamsMap = {
     source: [
       "https://api.binance.com/api/v3/avgPrice?symbol=BTCUSDT",
       "https://api.pro.coinbase.com/products/BTC-USD/trades?limit=1",
-      "https://api.kraken.com/0/public/Ticker?pair=BTCUSD",
+      "https://cors-anywhere.herokuapp.com/https://www.bitstamp.net/api/v2/ticker/btcusd",
```

Comment:
bitstamp API enforces strict CORS policy, need to use this proxy server to get around it